src/template/context.js

```diff
@@ -32,11 +32,12 @@ const context = {
             start: '\\s[\\s]+',
             end: '[(\n|\r\n)]+(\n|\r\n)?(?:[ \t].*?(?:(\n|\r\n)|$))*',
         },
-        filePath: 'file|path',
+        filePath: 'file|path|folder',
         delimiter: '-[^ \t(\n|\r\n)]+(\\s|=)[^ \t(\n|\r\n)-]',
         enumValues: {
-            enum: '<(([\\S]+\\|)+[\\S]+)>',
-            split: '|',
+            enum: '(<|\\(|\\")(([\\S]+(\\||\\",\\s\\"|\\sor\\s))+[\\S]+)' +
+            '(>|\\)|\\"|,\\s)',
+            split: /[\|]|\",\s\"|\"\sor\s\"/,
```

Comment:
I'm not sure about `or` char in this regex

Reply:
@svoboda-rabstvo check referenced issue #142
Enum may be as `formatter: "json", "string" or "verbose"`